code/src/components/MovieDetails.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/jsx-no-useless-fragment */
+/* eslint-disable react-hooks/exhaustive-deps */
+/* eslint-disable no-undef */
+import React, { useEffect, useState } from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
+import { MOVIEDETAILS_URL } from 'utils/urls'
+
+const MovieDetails = () => {
+  const [movieDetailList, setMovieDetailList] = useState([])
+  const { id } = useParams()
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    fetch(MOVIEDETAILS_URL(id))
+      .then((res) => res.json())
+      .then((data) => setMovieDetailList(data))
+  }, [id])
+
+  const goBack = () => {
+    navigate(-1)
+  }
+
+  return (
+    <section className="detail-page">
+      <div
+        className="background-img"
+        style={{ backgroundImage: `linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 1)), url(https://image.tmdb.org/t/p/original${movieDetailList.backdrop_path})` }}>
+        <button className="go-back-btn" type="button" onClick={goBack}>Go back to movies</button>
```

Comment:
you could replace this code snippet `<button className="go-back-btn" type="button" onClick={goBack}>Go back to movies</button>` with `<Link to="/" className="go-back-btn">
        <button>Go back to movies</button>
      </Link>` so that you have the Link usage instead